j2v/utils/helpers.py

```diff
@@ -123,8 +123,8 @@ def make_valid_variable_name(name):
 
 
 def camel_case_split(name):
-    splits = re.split('(?=[A-Z])', name)
-    if len(splits) == 0 or all(map(lambda c: c.isupper(), name)) or all(map(lambda c: c.islower(), name)):
+    splits = re.sub('(?!^)([A-Z][a-z]+)', r' \1', name).split()
```

Comment:
Regex hard hard to read and reverse-engineer. Can you add a comment what this line intends to do and which cases it addresses and which it might not for a conscious choice?

Reply:
Added comments to clarify this regex.